shopinvader/models/shopinvader_partner.py

```diff
@@ -106,4 +106,9 @@ def _prepare_create_child_params(self, parent, vals):
         if not v.get("type"):
             v["type"] = "other"
         v.pop("email")
+        # Remove field who doesn't exist in res.partner (partner_email,...)
```

Comment:
you might want to cherry pick this as well https://github.com/shopinvader/odoo-shopinvader/commit/9ad8da86f1ec0ae373e6a61eecb51c325bc750c5